frontend_app_file/src/container/File.jsx

```diff
@@ -1165,8 +1165,8 @@ export class File extends React.Component {
       state.content.current_revision_id,
       filenameWithoutExtension,
       state.fileCurrentPage,
-      500,
-      500
+      1024,
+      1024
```

Comment:
What are theses values? Default size of the preview?

Reply:
This is the size of the thumbnail requested to the backend; As said in the commit details, this could be improved in the future by defining steps by screen size for example, for mobile, 512x512